src/sign/types.ts

```diff
@@ -29,6 +29,19 @@ export interface Proposal {
   app?: string;
 }
 
+export interface UpdateProposal {
+  proposal: string;
+  from: string;
+  space: string;
+  timestamp: number;
+  type?: ProposalType;
+  title?: string;
+  body?: string;
+  discussion?: string;
+  choices?: string[];
+  plugins?: string;
```

Comment:
Are those type really optional ? Isn't the signing failing if they're missing ?

Reply:
this is resolved?

Reply:
I believe so